File: orderservice/src/main/java/com/orderservice/dto/CartDTO.java
package com.orderservice.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

import java.util.List;
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CartDTO {

    @JsonProperty("user_id")
    private Long userId;

    @JsonProperty("cart_items")
    private List<CartItemDTO> cartItems;

}

File: frontEnd/src/app/dto/cart/cart.dto.ts
import { CartItemDTO } from "./cartItem.dto";

export class UpdateCartDTO {    
    user_id: number;
    
    cart_items: CartItemDTO[];
      
    constructor(data: any) {
        this.user_id = data.user_id;
        this.cart_items = data.cart_items;    
    
    }
}